cdap-cli/src/main/java/co/cask/cdap/cli/command/LoadPreferencesCommand.java

```diff
@@ -63,16 +66,26 @@ public void perform(Arguments arguments, PrintStream printStream) throws Excepti
       throw new IllegalArgumentException("Not a file: " + file);
     }
 
-    if (contentType.isEmpty() || (!contentType.equals("json"))) {
-      throw new IllegalArgumentException("Unsupported file format. Only json format is supported");
+    if (contentType.isEmpty() || !(contentType.equals("json") || contentType.equals("xml"))) {
+      throw new IllegalArgumentException("Unsupported file format. Only json and xml formats are supported");
     }
 
     FileReader reader = new FileReader(file);
     Map<String, String> args = Maps.newHashMap();
     try {
-      args = GSON.fromJson(reader, MAP_STRING_STRING_TYPE);
+      if (contentType.equals("json")) {
+        args = GSON.fromJson(reader, MAP_STRING_STRING_TYPE);
+      } else {
```

Comment:
Do you think its better to explicitly put the if condition on this line, and throw an illegalArgumentException in the final `else` block?